### PR TITLE
Can service

### PR DESCRIPTION
**Pull request by BengtCruz**

### CODE REVIEW

File: can_service/can_service.cpp [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// DO NOT EDIT THIS FILE BY HAND!!
```
Comment (farochmehri) [inline can_service/can_service.cpp:1]: This file shall be generated by the script in the lib/canbus directory of the target repository

File: can_service/can_service.cpp [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+// Generated for node HUMAN MACHINE INTERFACE// Timestamp: 2020-10-31T01:33:31
```
Comment (farochmehri) [inline can_service/can_service.cpp:3]: The description shall be according to doxygen

File: can_service/can_service.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#ifndef CAN_SERVICE_H
```
Comment (farochmehri) [inline can_service/can_service.h:1]: Describe the file

File: can_service/can_signal.h [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#ifndef CAN_SIGNAL_H
```
Comment (farochmehri) [inline can_service/can_signal.h:1]: Describe the file

File: can_service/can_signal.h [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+#define CAN_SIGNAL_H
+
+#include <stdint.h>
+
+/**
+ * @brief Similiar to packing a message this function will write a signal from the 
+ * message index is given. 
+ * 
+ * @param msg_index
```
Comment (farochmehri) [inline can_service/can_signal.h:10]: Describe the parameters

File: can_service/service.py [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+module_path.mkdir(parents=True, exist_ok=True)
+shutil.copyfile("can_signal.h", "{0}/can_signal.h".format(temp))
+shutil.copyfile("can_service.h", "{0}/can_service.h".format(temp))
+
+
+# Message array for messages according to their getters and setters
+messages = []
+for message in canbus['messages']:
+    for signal in message['signals']:
+        if signal['getters'] == node or node in signal['setters']:
```
Comment (farochmehri) [inline can_service/service.py:62]: It shall be if node in signal['getters'] or node in signal['setters']:

**Review by farochmehri — CHANGES_REQUESTED** [review]
Read the reviews and fix the issues.

**Review by JesperMell — COMMENTED** [review]
This is some good Python!

**Pull request merged**